pyccl/cls.py

```diff
@@ -3,121 +3,105 @@
 from .core import check
 import numpy as np
 
-# Mapping between names for tracers and internal CCL tracer types
-tracer_types = {
-    'nc':               const.CL_TRACER_NC,
-    'number_count':     const.CL_TRACER_NC,
-    'wl':               const.CL_TRACER_WL,
-    'lensing':          const.CL_TRACER_WL,
-    'weak_lensing':     const.CL_TRACER_WL,
-    'cmbl':             const.CL_TRACER_CL,
-    'cmb_lensing':      const.CL_TRACER_CL,
-}
-
 # Same mapping for non-Limber integration methods
 nonlimber_methods = {
     'native': const.CCL_NONLIMBER_METHOD_NATIVE,
     'angpow': const.CCL_NONLIMBER_METHOD_ANGPOW,
 }
 
 function_types = {
-    'dndz':             const.CCL_CLT_NZ,
-    'bz':               const.CCL_CLT_BZ,
-    'bias':             const.CCL_CLT_BZ,
-    'sz':               const.CCL_CLT_SZ,
-    'm_bias':           const.CCL_CLT_SZ,
-    'rfz':              const.CCL_CLT_RF,
-    'red_fraction':     const.CCL_CLT_RF,
-    'baz':              const.CCL_CLT_BA,
-    'a_bias':           const.CCL_CLT_BA,
-    'wL':               const.CCL_CLT_WL,
-    'window_lensing':   const.CCL_CLT_WL,
-    'wM':               const.CCL_CLT_WM,
-    'window_magnif':    const.CCL_CLT_WM,
+    'dndz': const.CCL_CLT_NZ,
+    'bias': const.CCL_CLT_BZ,
+    'mag_bias': const.CCL_CLT_SZ,
+    'red_frac': const.CCL_CLT_RF,
+    'ia_bias': const.CCL_CLT_BA,
+    'lensing_win': const.CCL_CLT_WL,
+    'mag_win': const.CCL_CLT_WM,
```

Comment:
Or, otherwise, why did you not get rid of these?

Reply:
These are still used as arguments for a function.